docs/contributing/999-rfcs/2024-01-15-console-sign-in.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Wing Console Sign In"
+description: Requirements for Wing Console Sign In process
+---
+
+# Wing Console Sign In
+
+- **Author(s)**: @ainvoner
+- **Submission Date**: 2024-01-15
+- **Stage**: Proposal
+- **Stage Date**: 2024-01-15
+
+This document describes the Wing Console Sign In process and rationale.
+
+## Background
+
+As we seek to find **Sales and Marketing Opportunities**, it has become increasingly important to establish a more direct and meaningful connection with our users.
+One of the ways to achieve this is by implementing a sign-in flow within the Wing Console application.
+
+## Requirements
+
+New users of Wing Console will be required to sign in with their GitHub account before having access to the Wing Console application.
```

Comment:
The more enterprisier the audience, the more likely it'll be that people might not have a Github account. Any chance to track this somehow?

Reply:
How hard is it to add additional providers? If not too difficult then maybe we should.

Alternatively, we can add a "I don't have a GitHub account" link which will skip sign in and we can track analytics.

Reply:
if we will allow to skip sign - no one will sign in.
I think, for initial phase, support only GitHub (we can add additional sign in providers later).

Reply:
+1

Reply:
We would need to do some refactor in Wing Cloud to allow other providers. It'd take some time right now.

> Alternatively, we can add a "I don't have a GitHub account" link which will skip sign in and we can track analytics.

That's what we do now, right? Just anonymous analytics.